set.mm

```diff
@@ -98434,6 +98434,345 @@ a Cantor normal form (and injectivity, together with coherence
+       26-Jun-2015.) $)
+    dftrpred4g $p |- ( ( X e. A /\ R Se A ) ->
+        TrPred ( R , A , X ) =
+        U_ y e. Pred ( R , A , X ) ( { y } u. TrPred ( R , A , y ) ) ) $=
+      ( wcel wse wa ctrpred cpred cv ciun cun csn dftrpred3g iunun iunid uneq1i
+      eqtri eqtr4di ) DBEBCFGBCDHBCDIZATBCAJZHZKZLZATUAMZUBLKZABCDNUFATUEKZUCLU
+      DATUEUBOUGTUCATPQRS $.
+  $}
+
+  ${
+    $d A y $.  $d R y $.  $d X y $.
+    $( If ` R ` partially orders ` A ` , then the transitive predecessors are
+       the same as the immediate predecessors .  (Contributed by Scott Fenton,
+       28-Apr-2012.)  (Revised by Mario Carneiro, 26-Jun-2015.) $)
+    trpredpo $p |- ( ( R Po A /\ X e. A /\ R Se A ) ->
+                      TrPred ( R , A , X ) = Pred ( R , A , X ) ) $=
+      ( vy wpo wcel wse w3a ctrpred cpred cv wss simp2 simp3 wa predpo ralrimiv
+      wral 3adant3 cvv ssidd trpredmintr syl22anc setlikespec syl 3adant1 eqssd
+      trpredpred ) ABEZCAFZABGZHZABCIZABCJZULUJUKABDKZJUNLZDUNRZUNUNLUMUNLUIUJU
+      KMUIUJUKNUIUJUQUKUIUJOUPDUNABCUOPQSULUNUADAUNBCUBUCUJUKUNUMLZUIUJUKOUNTFU
+      RABCUDATBCUHUEUFUG $.
+  $}
+
+  ${
+    $d R a i j y $.  $d X a i j y $.
+    $( The class of transitive predecessors is empty when ` A ` is empty.
+       (Contributed by Scott Fenton, 30-Apr-2012.) $)
+    trpred0 $p |- TrPred ( R , (/) , X ) = (/) $=
+      ( vi va vy vj c0 com cv cvv cpred ciun cmpt crdg cres cfv wcel wceq pred0
+      iuneq2i ctrpred dftrpred2 iun0 eqtri mpteq2i rdgeq12 mp2an reseq1i fveq1i
+      a1i csuc wrex nn0suc fveq2 0ex fr0g ax-mp eqtrdi nfcv eqid eqidd frsucmpt
+      wo mpan2 fveqeq2 syl5ibrcom rexlimiv jaoi syl syl5eq 3eqtri ) GABUACHCIZD
+      JEDIZGAEIZKZLZMZGABKZNZHOZPZLCHGLGEGACBDUBCHWAGVLHQZWAVLDJGMZGNZHOZPZGVLV
+      TWEVSWDHVQWCRVRGRVSWDRDJVPGVPEVMGLGEVMVOGVOGRVNVMQAVNSUJTEVMUCUDUEABSVRGV
+      QWCUFUGUHUIWBVLGRZVLFIZUKZRZFHULZVCWFGRZFVLUMWGWLWKWGWFGWEPZGVLGWEUNGJQZW
+      MGRUOGJWCUPUQURWJWLFHWHHQZWLWJWIWEPGRZWOWNWPUODGWHGGWEJDGUSZDWHUSWQWEUTVM
+      WHWEPRGVAVBVDVLWIGWEVEVFVGVHVIVJTCHUCVK $.
+  $}
+
+  ${
+    $d R a y $.  $d A a y $.  $d X a y $.
+    $( The transitive predecessors of a relation form a set (NOTE: this is the
+       first theorem in the transitive predecessor series that requires
+       infinity).  (Contributed by Scott Fenton, 18-Feb-2011.) $)
+    trpredex $p |- TrPred ( R , A , X ) e. _V $=
+      ( va vy ctrpred cvv cpred ciun cmpt crdg com cres crn cuni df-trpred wcel
+      cv wfn frfnom omex fnex mp2an rnex uniex eqeltri ) ABCFDGEDRABERHIJZABCHZ
+      KLMZNZOGEABCDPUJUIUILSLGQUIGQUHUGTUALGUIUBUCUDUEUF $.
+  $}
+
+  ${
+    $d A z $.  $d R z $.  $d X z $.  $d Y z $.  $d A a i j y z $.
+    $d R a i j y $.  $d X a i j y $.  $d Y a i j y $.
+    $( If ` Y ` is an ` R ` , ` A ` transitive predecessor, then it is either
+       an immediate predecessor or there is a transitive predecessor between
+       ` Y ` and ` X ` .  (Contributed by Scott Fenton, 9-May-2012.)  (Revised
```

Comment:
```suggestion
    $( A transitive predecessor of ` X ` is either an immediate predecessor of
       ` X ` or an immediate predecessor of a transitive predecessor of ` X ` .
       (Contributed by Scott Fenton, 9-May-2012.)  (Revised
```

Note: it may be worth stating it as a biconditional.

Note: if this does not require ax-inf, as I think is the case, then you can move it before trpredex.